minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloConfig.java

```diff
@@ -255,4 +255,17 @@ public MiniAccumuloConfig setNativeLibPaths(String... nativePathItems) {
     impl.setNativeLibPaths(nativePathItems);
     return this;
   }
+
+  /**
+   * Sets the classpath elements to use when spawning processes.
+   *
+   * @param classpathItems
+   *          the classpathItems to set
+   * @return the current instance
+   * @since 2.0.0
+   */
+  public MiniAccumuloConfig setClasspathItems(String... classpathItems) {
```

Comment:
Could call this `setClasspath` and drop `Items`.  I don't think `Items` adds any useful information.

Reply:
Pushed an update for this